File: android/PureOSLaunchCandidate/app/src/main/java/com/pureos/mobilecore/v1743/MainActivity.java
package com.pureos.mobilecore.v1743;

import android.os.Bundle;
import android.widget.TextView;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        TextView status = findViewById(R.id.pureosStatus);
        status.setText("PureOS Mobile Core v17.43\nDebug launch candidate online\n\nPure Intelligence route ready\nPureLang layer ready\nApproval gate active\nGitHub Actions APK artifact ready");
    }
}

File: android/PureOSLaunchCandidate/app/src/main/java/com/pureos/mobilecore/v1742/MainActivity.kt
package com.pureos.mobilecore.v1742

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        findViewById<TextView>(R.id.pureosStatus).text = buildString {
            appendLine("PureOS Mobile Core v17.42")
            appendLine("Debug launch candidate online")
            appendLine()
            appendLine("Pure Intelligence: local runtime route ready")
            appendLine("PureLang: command layer seed ready")
            appendLine("Governor: approval-first safety gate active")
            appendLine("GitHub Actions: APK artifact verified")
        }
    }
}
